test/integration/default/verify/controls/aws_flow_log.rb

```diff
@@ -0,0 +1,95 @@
+iam_fixtures = {}
+[
+  'flow_log_default_subnet_id',
+  'flow_log_default_vpc_id',
+  'log_metric_filter_1_log_group_name'
+].each do |fixture_name|
+  iam_fixtures[fixture_name] = attribute(
+  fixture_name,
+  default: "default.#{fixture_name}",
+  description: 'See ../build/iam.tf',
+  )
+end
+
+ec2_fixtures = {}
```

Comment:
You needn't have two blocks of fixtures - they all read from the same attributes file.